d2oracle/edit_test.go

```diff
@@ -2001,6 +2001,32 @@ c: {
 				assert.JSON(t, len(g.Objects), 3)
 			},
 		},
+		{
+			name: "underscore-connection",
+
+			text: `a: {
+  b
```

Comment:
For the test case, I don't see how your fix applies. Wouldn't this have been the first reference anyway?

Reply:
i guess not. idk how they're ordered

Reply:
i add this line

<img width="532" alt="Screen Shot 2023-02-14 at 5 06 11 PM" src="https://user-images.githubusercontent.com/3120367/218898794-cbd40ae1-2fd3-497f-8d16-4d2248d5d181.png">

and get this
<img width="401" alt="Screen Shot 2023-02-14 at 5 06 05 PM" src="https://user-images.githubusercontent.com/3120367/218898803-490a2e40-c75d-4802-9f97-e5b9c47eae5e.png">

Reply:
Ohh that's cause this function is being called for `c` not `a.b` lol.

I don't think it's related to underscores though, you could remove the underscores and still reproduce this I believe.